client/src/components/chat/ChatTile.tsx

```diff
@@ -1,35 +1,25 @@
-import styles from '../styles/ChatTile.module.css'
-import {ReactComponent as ReadIcon} from '../assets/message_read.svg';
-import {ReactComponent as UnreadIcon} from '../assets/message_unread.svg';
-import { ChatData } from '../adapters/interfaces/IChatDataAdapter.ts'
+import styles from '../styles/ChatTile.module.css';
+import { ReactComponent as ReadIcon } from '../assets/message_read.svg';
+import { ReactComponent as UnreadIcon } from '../assets/message_unread.svg';
+import { ChatData } from '../adapters/interfaces/IChatDataAdapter.ts';
 
-const ChatTile = ({
-  chat,
-  setSelectedChat,
-}: {
-  chat: ChatData
-  setSelectedChat: (value: ChatData) => void
-}) => {
+const ChatTile = ({ chat }: { chat: ChatData }) => {
   const {
     lastMessage,
-    lastMessageSenderName,
-    lastMessageStatus,
+    lastSender,
+    isLastMessageRead,
     chatName,
     profilePicture,
-    id,
-  } = chat
+  } = chat;
 
   const messageToDisplay =
```

Comment:
This is redundant as I have added truncate to the <div> containing all of the texts